### ISSUE TITLE
How we keep Giveth Planning organized! (READ ME)

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#373]
Hey Comms Stars and Unicorns-to-be!

We use the Giveth Planning Board to organize issues related to Giveth Communications. This issue outlines our process for zenhub coodination and QA collaboration.

## Zenhub Coordination
- Issues move from "New" to "Backlog" once they assigned to someone or picked up
- Issues move from "Backlog" to "Sprint Backlog" when we decided to take them on within the next 1-2 weeks
- Issues move from "Spring Backlog" to "In Progress" when they are being actively worked on. There should only be 1-2 issues per assignee in "In Progress"
- Add "dependencies" when an issues is blocked by another issue
- Use the <blocked> label when an issues completion is being blocked by something that is not a dependency, and leave a comment explaining the blocker
     - Blocked issues that are otherwise competed but waiting for something to go live can live in "done"
     - Blocked issues that are waiting before they can be started can live in "backlog"
- Issues are arrange in order of priority: highest priority are at the top
- Assignees are responsible for updating their issues according to progress

## QA and collaboration
- Documentation can generally be drafted in HackMD or Medium (wherever appropriate) to ensure correct formatting
- When a piece is ready for QA, the owner can copy/paste into google docs and add that shareable link into the issue, and tag people for collaborative QA
- The owner can then incorporate the adds and copy/paste it back into hackMD or Medium and make final formatting adjustments
- We should have 2 Comms Stars QAing each pieces to ensure consistent voice
